pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -32,19 +32,21 @@ import (
 var _ ConnTrackDumper = new(connTrackSystem)
 
 type connTrackSystem struct {
-	nodeConfig  *config.NodeConfig
-	serviceCIDR *net.IPNet
-	connTrack   NetFilterConnTrack
+	nodeConfig    *config.NodeConfig
+	serviceCIDR   *net.IPNet
+	isAntreaProxy bool
```

Comment:
`withAntreaProxy` or `antreaProxyEnabled` make more sense as names for this

Reply:
Went with `isAntreaProxyEnabled`. Hope that sounds good.